keyboards/pearlboards/pearl/readme.md

```diff
@@ -0,0 +1,15 @@
+# Pearl
+
+![Pearl](https://i.imgur.com/xdEtT7uh.jpg)
+
+Pearl is a future-proof premium 40%. This is the second round of it, reworked for 2021. This firmware is for the latest QMK PCB's.
+
+* Keyboard Maintainer: Koobaczech
+* Hardware Supported: Pearl
+* Availability: https://pearlboards.net/products/pearl-40
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```